### ISSUE TITLE
UCUM validation by CommonCodeSystemsTerminologyService fails

### ISSUE THREAD

**Issue by arminShv** [hapifhir/hapi-fhir#4559]
Hello,

It seems that there is a problem with UCUM validation when using the validator CommonCodeSystemsTerminologyService (see [here](https://hapifhir.io/hapi-fhir/docs/validation/validation_support_modules.html#commoncodesystemsterminologyservice)).

**To Reproduce**
Try to create the following example resource by a POST-Request.

```
{
  "resourceType": "Observation",
  "id": "example",
  "status": "final",
  "code": {
    "coding": [
      {
        "system": "http://loinc.org",
        "code": "11555-0",
        "display": "Base excess in Blood by calculation"
      }
    ]
  },
  "valueQuantity": {
    "value": 12.6,
    "unit": "mmol/l",
    "system": "http://unitsofmeasure.org",
    "code": "mmol/L"
  }
}
```

The response is:
![image](https://user-images.githubusercontent.com/29888058/219385087-21dc0f65-9d33-4428-9fc3-4bc5e48743d0.png)

**Expected behavior**
The codes mentioned in [this](http://hl7.org/fhir/R4/valueset-ucum-units.html) value set are expected to pass the validation by CommonCodeSystemsTerminologyService.

**Environment**
 - HAPI FHIR Version 5.7.0
 - OS: Windows 11
 - ucum-1.0.3.jar library

**Comment by laneb** [comment]
Encountered similar (all codes attributed to system `http://unitsofmeasure.org` rejected) and resolved by deleting the CodeSystem with that url. Appears that `CodeSystem` is created when `hapi-fhir-starter-server` installs conformance resources, but it is empty (no codes) on creation. Then I suppose `hapi-fhir` defaults to validating UCUM codes against that empty CodeSystem instead of using the `CommonCodeSystemsTerminologyService`.

**Comment by jamesagnew** [comment]
This should be resolved by #5271, which now gives `CommonCodeSystemsTerminologyService` priority over locally installed CodeSystem resources. Please comment and/or reopen if this still occurs against latest master.

**Comment by laneb** [comment]
Thanks @jamesagnew!
